test/contract-factory-test.ts

```diff
@@ -4,4 +4,8 @@ describe("ContractFactory", () => {
     it("should be created", async () => {
         await starknet.getContractFactory("contract");
     });
+
+    it('load artifact outside artifacts folder', async () => {
+        await starknet.getContractFactory("../test/test-aritfacts/contract");
```

Comment:
I'd move this to `path-test.ts` or not have it at all, see the other comment.

Reply:
Removed it